models/app-models.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection.js");
+
+exports.selectTopics = async () => {
+  const topics = await db.query("SELECT * FROM topics;");
+    return topics.rows;
+};
+
+exports.selectArticle = async (article_id) => {
```

Comment:
Make sure you write your tests first before writing the endpoint. Your code should be written according to what is expected by the tests :)